rancher_pr_deploy.sh

```diff
@@ -17,34 +23,49 @@ if [[ -n "$TRAVIS" ]]; then
   # IF PULL REQUEST
   if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then
 
-    install_rancher
+    if [ ! "$(command -v rancher)" ]
+    then 
+      install_rancher
+    fi
 
-    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/${RANCHER_TEMPLATE_NAME}/0/docker-compose.yml"  -o docker-compose.yml
-    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/master/templates/${RANCHER_TEMPLATE_NAME}/0/rancher-compose.yml" -o rancher-compose.yml
+    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/feature/changes-for-ci-script/templates/${RANCHER_TEMPLATE_NAME}/0/docker-compose.yml"  -o docker-compose.yml
+    curl -s "https://raw.githubusercontent.com/nhsuk/nhsuk-rancher-templates/feature/changes-for-ci-script/templates/${RANCHER_TEMPLATE_NAME}/0/rancher-compose.yml" -o rancher-compose.yml
 
-    touch answers.txt
-    echo -n "" > answers.txt
+    # need two args for the split but ORG is not needed
+    # shellcheck disable=SC2034 
+    IFS=/ read -r ORG REPO <<< "${TRAVIS_REPO_SLUG}"
 
-    {
-      echo "traefik_domain=dev.c2s.nhschoices.net"
-      echo "profiles_docker_image_tag=pr-${TRAVIS_PULL_REQUEST}"
-      echo "splunk_hec_endpoint=https://splunk-collector.cloudapp.net:8088"
-      echo "splunk_hec_token=${SPLUNK_HEC_TOKEN}"
-      echo "hotjar_id=265857"
-      echo "google_id=UA-67365892-5"
-      echo "webtrends_id=dcs222rfg0jh2hpdaqwc2gmki_9r4q"
-    } >> answers.txt
+    cat <<EOT  > answers.txt
+traefik_domain=dev.c2s.nhschoices.net
+$(create_parameter_name "$REPO")_docker_image_tag=pr-${TRAVIS_PULL_REQUEST}
+splunk_hec_endpoint=https://splunk-collector.cloudapp.net:8088
+splunk_hec_token=${SPLUNK_HEC_TOKEN}
+hotjar_id=265857
+google_id=UA-67365892-5
+webtrends_id=dcs222rfg0jh2hpdaqwc2gmki_9r4q
+EOT
+ 
+    if [ -n "$DEPENDANT_SERVICE" ]
```

Comment:
I'm expecting that when we introduce the api with mongo then, rather than just handling a single dependant service we will change this to loop through multiple ones.